oauth2client/client.py

```diff
@@ -1222,6 +1229,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        # This import needs to be here to avoid a circular dependency.
```

Comment:
-1 on TODO's in code. That is the value of having a world class issue tracking system